ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -265,6 +265,8 @@ public void processMessage(final Capability cap, final Message message) {
 
     if (this.mergePeerFilter.isPresent()) {
       if (this.mergePeerFilter.get().disconnectIfGossipingBlocks(message, ethPeer)) {
+        LOG.info(
+            "here we are disconnecting at request from mergePeerFilter disconnectIfGossipingBlocks");
```

Comment:
not a bad log line to have, but "here we are" sounds like a comment rather than a log line.  This log line might be more helpful if we listed the enode and/or peer id of the peer we are disconnecting

Reply:
done